cmd/gengo/gen.go

```diff
@@ -88,6 +89,10 @@ func (p *Runner) GenGo(dir string, recursive bool, base *cl.Config) {
 	var gopTime time.Time
 	var gogenTime time.Time
 	var pkgFlags int
+	if modload.ClassModFile != nil && modload.ClassModFile.Classfile != nil {
```

Comment:
`Runner` is not only command line tools, but also a utility package. So, don't let it depend global states.